config/model_configs/single_column_precipitation_test.yml

```diff
@@ -0,0 +1,34 @@
+config: "column"
+initial_condition: "PrecipitatingColumn"
+surface_setup: "DefaultExchangeCoefficients"
+z_elem: 200
+z_max: 10000.0
+z_stretch: false
+dt: "10secs"
+t_end: "1500secs"
+dt_save_to_disk: "500secs"
+moist: "nonequil"
+precip_model: "1M"
+precip_upwinding: "first_order"
+vert_diff: "true"
+hyperdiff: "false"
+rayleigh_sponge: true
+regression_test: false
+check_precipitation: true
+job_id: "single_column_precipitation_test"
+toml: [toml/single_column_precipitation_test.toml]
+diagnostics:
+  - short_name: hus
+    period: 500secs
+  - short_name: clw
+    period: 500secs
+  - short_name: cli
+    period: 500secs
+  - short_name: husra
+    period: 500secs
+  - short_name: hussn
+    period: 500secs
+  - short_name: ta
+    period: 500secs
+  - short_name: wa
+    period: 500secs
```

Comment:
Thanks! For some reason I thought that NetCDF is not available for use yet. I'll switch it over. This will also simplify the plotting

Reply:
The NetCDF writer that is currently in `main` has some limitations, but the complete one should be around the corner.

Reply:
Actually, now I remember. I first tried running with netcdf output, but it's currently only supported on cubed spheres. The test I'm adding is a single column simulation, so I'll have to stick with hdf for now

Reply:
You can start using directly PR #2179. The PR is ClimaCore has been merged and a new version of ClimaCore has been released, so that PR #2179 should be merged very soon hopefully!